src/api/category/category.resolver.ts

```diff
@@ -0,0 +1,26 @@
+import { Resolver, Query, Args, Mutation } from '@nestjs/graphql';
+import { Category, CategorySelect } from './model';
+import { CategoryArgs, CategoryCreateInput } from './dto';
+import { CategoryService } from './category.service';
+import { GraphQLFields, IGraphQLFields } from '../../shared/decorators';
+
+@Resolver(() => Category)
+export class CategoryResolver {
+  constructor(private readonly categoryService: CategoryService) {}
+
+  @Query(() => Category,{nullable:true})
+  public async category(
```

Comment:
In order to create a Castegory, we need to have an account, so let's add the auth decorator